File: com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IBannerEventListener.java
package com.chartboost.heliumsdk.unity;

/**
 * Interface for sending real-time Banner events
 */
public interface IBannerEventListener {
    void DidLoadBanner(String placementName, int errorCode, String errorDescription);
    void DidClickBanner(String placementName, int errorCode, String errorDescription);
    void DidRecordImpression(String placementName, int errorCode, String errorDescription);
    void DidWinBidBanner(String placementName, String auctionId, String partnerId, double price);
}

File: com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IBannerEventListener.kt
package com.chartboost.heliumsdk.unity

/**
 * Interface for sending real-time Banner events
 */
interface IBannerEventListener {
    fun DidLoadBanner(placementName: String, loadId: String, error: String)
    fun DidClickBanner(placementName: String)
    fun DidRecordImpression(placementName: String)
    fun DidWinBidBanner(placementName: String, auctionId: String, partnerId: String, price: Double, error: String)
}
